app/assemblers/assemble_exercise_representations_admin.rb

```diff
@@ -0,0 +1,37 @@
+class AssembleExerciseRepresentationsAdmin
+  include Mandate
+
+  def self.keys = %i[page order criteria track_slug]
+
+  initialize_with :params
+
+  def call
+    SerializePaginatedCollection.(
+      representations,
+      serializer: SerializeExerciseRepresentations,
+      serializer_kwargs: { params: },
+      meta: {
+        unscoped_total: Exercise::Representation.with_feedback.count
+      }
+    )
+  end
+
+  private
+  memoize
+  def representations
+    Exercise::Representation::Search.(
+      mentor: nil,
```

Comment:
Let's change this so that we don't need to express it. (ie change the initializer to have `nil` as the default)